src/components/flat-action-button/FlatActionButton.tsx

```diff
@@ -6,11 +6,21 @@ export type IFlatActionButtonType = "white" | "blue" | "success" | "danger" | "g
 interface IFlatActionButton {
   type: IFlatActionButtonType;
   onClick?: () => void;
+  width?: string;
+  height?: string;
+  disabled?: boolean;
 }
 
-export const FlatActionButton: React.FC<IFlatActionButton> = ({ type, onClick, children }) => {
+export const FlatActionButton: React.FC<IFlatActionButton> = ({
+  type,
+  width,
+  height,
+  onClick,
+  disabled,
+  children,
+}) => {
   return (
-    <SFlatActionButton type={type} onClick={onClick}>
+    <SFlatActionButton disabled={disabled} type={type} width={width} height={height} onClick={onClick}>
```

Comment:
@bredek this shouldn't be changed like this, otherwise we are breaking the definitions in DS. Why do we need this?

Reply:
Looks like the component should use a different component. https://www.figma.com/file/1hz5D4Q7pj5w0YrRw7hXbu/UI-Kit?node-id=887%3A13505

Reply:
this component should be using Sticked button, not this one, and also the changes to the flat action button should be reverted.

![Uploading Screen Shot 2022-01-06 at 10.55.05 AM.png…]()

Reply:
But we don't have any sticked button in our library.
We had a talk regarding this button with team.

![image](https://user-images.githubusercontent.com/14944987/148399779-d3ba9e34-ff83-472e-b268-2d828a98835f.png)

Reply:
Yes, it is in flight https://linear.app/caisy/issue/CY-632/buttons-bar

Reply:
https://league-9zpvcb3b8-caisy.vercel.app/?path=/story/components-buttons-button--default&args=content:DEFAULT;sticked:true

Reply:
https://linear.app/caisy/issue/CY-632/buttons-bar Already merged into dev